__tests__/app.test.js

```diff
@@ -165,3 +166,85 @@ describe("PATCH article", () => {
       });
   });
 });
+
+describe("GET articles", () => {
+  test("200: get all articles including property comment_count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles.length > 0).toBe(true);
+
+        body.articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("200: get all articles by descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles.length > 0).toBe(true);
```

Comment:
Same length comment as above - should be more specific

Reply:
I'd also argue though, if you are doing exactly the same request as the above test then you don't need to check the length or that it's an array because it's covered in the above test block (line 171)